auditlog/__openerp__.py

```diff
@@ -21,7 +21,7 @@
 
 {
     'name': "Audit Log",
-    'version': "8.0.1.2.0",
+    'version': "8.0.1.3.0",
```

Comment:
I checked this document, and as it is a non-breaking new feature which does not require a data migration, I opted for `1.3.0`. This feature is less intrusive than #357 where I have done the same incrementation. But it does not matter for me if it has to be `2.0.0`, what other peoples think?